code/modules/research/designs/weapon_designs.dm

```diff
@@ -237,25 +229,23 @@
 	category = list("Weapons")
 
 /datum/design/xray
-	name = "Xray Laser Gun"
+	name = "Xray Laser Gun Parts Kit"
 	desc = "Not quite as menacing as it sounds"
```

Comment:
Might want to update this desc to note that it's a kit, as well as what the gun actually does!